src/components/builder/i18n.tsx

```diff
@@ -12,6 +12,7 @@ import {AnyComponentSchema} from '@/types/schemas';
 import {Component, TextField} from '../formio';
 import ComponentLabel from '../formio/component-label';
 import './i18n.scss';
+import './table.scss';
```

Comment:
since this isn't really tied to a react component, maybe we should perform this import once in the `index.ts` of the library? I'm not sure what the right approach is here

Possibly we can also use `@use './table';` in the `i18n.scss` module to ensure it's loaded :thinking:

Reply:
Mhh in the `index.ts` might be better yeah.. That we treat this as global styling. Because, indeed, it currently isn't tied to any component.

Reply:
reverted because storybook itself doesn't hit the `index.ts` and then the styles are absent in the chromatic snapshots